scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/QuoteVerifier.scala

```diff
@@ -16,11 +16,25 @@
 
 package com.intel.analytics.bigdl.ppml.attestation
 
+import org.apache.logging.log4j.LogManager
 
 abstract class QuoteVerifier {
```

Comment:
My fault. Need to extend QuoteVerifier and move code to that class. 
SGXDCAPQuoteVerifier